### ISSUE TITLE
alwasy catches cancel

### ISSUE THREAD

**Issue by madzhuga** [Godofbrowser/vuejs-dialog#16]
Hi! Thanks for a great component. I have a question
```      
this.$dialog.confirm('Are you sure?', options)
      .then(function () {
        console.log('Clicked ok')
      })
      .catch(function () {
        console.log('Clicked cancel')
      });
```
No matter which button I click it always adds 'Clicked cancel' to a log.

**Comment by Godofbrowser** [comment]
It could be that there's an error in the `then` handler. Try logging the error to see what's actually causing it. You can also paste it here.

```
.catch(function (error) { console.log(error) });
```

**Comment by Godofbrowser** [comment]
You're welcome. Glad it's working now.

Although i don't think that could have been the reason.

Importing with a custom name is actually allowed. The module was exported as a default so any name goes.
it could have been some syntax error, or the name used conflicts with something else.

But all the same, glad it's working now.

**Comment by madzhuga** [comment]
yeah, I already found a reason - just checked error in catch - it was catching 'Ok' properly but then failed to run my code - and then .catch catched my own error.
